app/views/News.js

```diff
@@ -135,7 +136,7 @@ class NewsScreen extends Component {
                 sliderWidth={width}
                 itemWidth={width * 0.85}
                 layout={'default'}
-                scrollEnabled
+                scrollEnabled={this.state.newsUrls.length == 1 ? false : true} // ScrollEnabled will be false if single url.
```

Comment:
could you explain why scroll is only being enabled in this scenario? I'm a little confused about why it corresponds to  `newsUrl.length` .

Reply:
also can be simplified to 

```js
 scrollEnabled={!this.state.newsUrls.length === 1}
```

Reply:
Seems #730  issue in iOS, display this blue screen continuously unless user swipe on the blue screen when single carousel view. Scroll will be disable if newsUrl.length is one.